src/app/(proper_react)/(redesign)/(authenticated)/admin/emails/EmailTrigger.tsx

```diff
@@ -23,6 +24,7 @@ export const EmailTrigger = (props: Props) => {
     props.emailAddresses[0],
   );
   const [isSendingVerification, setIssSendingVerification] = useState(false);
+  const [isSendingBreachAlert, setIssSendingBreachAlert] = useState(false);
```

Comment:
```suggestion
  const [isSendingBreachAlert, setIsSendingBreachAlert] = useState(false);
```

Should this be `setIsSendingBreachAlert`?

Reply:
Haha, looks like that wasn't the only variable I made that typo for: 26e430156